packages/braintree-utils/src/braintree-integration-service.spec.ts

```diff
@@ -359,6 +385,39 @@ describe('BraintreeIntegrationService', () => {
 
             expect(braintreeIntegrationService.mapToLegacyShippingAddress(props)).toEqual(expects);
         });
+
+        it('returns empty string if name is empty', () => {
+            const details = {
```

Comment:
Could you please use already declare the similar variable (detailsMock) in upper scope?

And take info from it for the expecting result value

Reply:
I created test scope variable with different fields values, that's why i didn't use existing one